src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py

```diff
@@ -96,14 +95,14 @@ def unregister():
     if session.pop('registered', None):
         task = tasks.pop(session.sid)
         flask_socketio.disconnect(task.websocket_id, namespace="/results")
-        if gstreamer:
+        if hasattr(task, 'port'):
```

Comment:
asi by bylo dobré tenhle test založit na něčem jiném, např. issubclass(type(task), TaskHandlerGstreamer)?

Reply:
Zjednodušeně  `isinstance(task, TaskHandlerGstreamer)`?

Reply:
Jo, to je lepší, nevěděl sem že isinstance funguje i obecně na předky a ne jen na konkrétní class..